products/views.py

```diff
@@ -1,3 +1,73 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Q
+
+from datetime       import datetime
+from .models        import Category, Subcategory, Product
+from users.models   import User, Host
+
+class ProductView(View): 
```

Comment:
List를 반환하는 경우에는 `ProductsView` 혹은 `ProductListView`로 변수명을 정하는게 Django View 네이밍 컨벤션입니다~
```suggestion
class ProductsView(View): 
```

Reply:
ProductListView로 수정했습니다~!